frcxx_nft/src/nft/state.rs

```diff
@@ -0,0 +1,98 @@
+use cid::multihash::Code;
+use cid::Cid;
+use fvm_ipld_amt::Amt;
+use fvm_ipld_amt::Error as AmtError;
+use fvm_ipld_blockstore::Block;
+use fvm_ipld_blockstore::Blockstore;
+use fvm_ipld_encoding::tuple::*;
+use fvm_ipld_encoding::CborStore;
+use fvm_ipld_encoding::DAG_CBOR;
+use fvm_shared::ActorID;
+use thiserror::Error;
+
+pub type TokenID = u64;
+
+#[derive(Serialize_tuple, Deserialize_tuple, Debug)]
+pub struct BatchMintReturn {
+    pub tokens: Vec<TokenID>,
```

Comment:
The Bitfield structure will really come into its own here, I recommend we use it (and spec it to be used in the FRC)